src/config.ts

```diff
@@ -7,5 +7,5 @@ dotenv.config({ path: '.env' });
 
 export const fallbackImages = {
   avatar:
-    'https://daily-now-res.cloudinary.com/image/upload/t_logo,f_auto/v1664367305/placeholders/placeholder3',
+    'https://daily-now-res.cloudinary.com/image/upload/f_auto/v1664367305/placeholders/placeholder3',
```

Comment:
This also needs to happen in the webapp though for the email flow, would you mind doing it there as well? 🙏

Reply:
Already did in my other PR :)